test/resources/launchDir/User/settings.json

```diff
@@ -1,5 +1,6 @@
 {
     "mssql.openQueryResultsInTabByDefaultDoNotShowPrompt": false,
-    "mssql.enableRichExperiences": false,
+    "mssql.enableRichExperiences": true,
+    "mssql.enableNewConnectionFeature": false,
```

Comment:
We are not enabling new connection features?

Reply:
err, what _is_ `enableNewConnectionFeature`?